labrad/units.py

```diff
@@ -268,63 +268,71 @@ def units(self):
         """A string representation of the unit of this value."""
         return self.unit.name
 
-    def _sum(self, other, sign1, sign2):
+    def _convert_units(self, other):
+        """Convert two objects to use compatible units.
```

Comment:
Might want to point out in the doc that we return `Unit('')` if self is dimensionless.